dftimewolf/lib/processors/turbinia_gcp.py

```diff
@@ -62,7 +62,9 @@ def SetUp(self,
     if disk_names:
       for disk in disk_names.strip().split(','):
         if disk:
```

Comment:
Guard clause here as well to prevent a third indentation

```suggestion
        if not disk:
          continue
```